cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go

```diff
@@ -99,21 +99,17 @@ func TestValidateTokenGroups(t *testing.T) {
 func TestValidateNodeRegistrationOptions(t *testing.T) {
 	var tests = []struct {
 		nodeName       string
-		criSocket      string
 		expectedErrors bool
 	}{
-		{"", "/some/path", true},                                                             // node name can't be empty
-		{"INVALID-NODENAME", "/some/path", true},                                             // Upper cases is invalid
-		{"invalid-nodename-", "/some/path", true},                                            // Can't have trailing dashes
-		{"invalid-node?name", "/some/path", true},                                            // Unsupported characters
-		{"valid-nodename", "/some/path", false},                                              // supported
-		{"valid-nodename-with-numbers01234", "/some/path/with/numbers/01234/", false},        // supported, with numbers as well
```

Comment:
your refactor dropped this test case, but overall some of the node name cases don't matter much as IsDNS1123Subdomain is unlikely to change much anytime soon.

Reply:
Right. This is again along the same lines as the previous conversation. All we really need here, IMHO, is 1 test case that will generate an error in `IsDNS1123Subdomain`.